packages/math-inline/src/__tests__/main.test.js

```diff
@@ -24,53 +54,181 @@ xdescribe('Main', () => {
             type: 'none',
             default: 'Incorrect'
           }
-        }
+        },
+        customKeys: [
+          '\\left(\\right)',
+          '\\frac{}{}',
+          'x\\frac{}{}'
+        ]
       }
-    },
-    onSessionChange: jest.fn(),
-    session: {}
+    }
   };
 
   let wrapper;
   let component;
 
   beforeEach(() => {
-    wrapper = shallowChild(Main, defaultProps, 1);
+    wrapper = shallow(<Main {...defaultProps} />);
+    component = shallowChild(Main, defaultProps, 1);
   });
 
-  it('renders correctly', () => {
-    component = wrapper();
-
-    expect(component.find(CorrectAnswerToggle).length).toEqual(1);
-    expect(component.find(Feedback).length).toEqual(0);
-    //
-    // expect(component.state()).toEqual({
-    //   session: {
-    //     answers: {
-    //       1: [false, false],
-    //       2: [false, false],
-    //       3: [false, false],
-    //       4: [false, false]
-    //     }
-    //   },
-    //   showCorrect: false,
-    //   shuffledRows: [{
-    //     id: 1,
-    //     title: 'Question Text 1',
-    //     values: [false, false]
-    //   }, {
-    //     id: 2,
-    //     title: 'Question Text 2',
-    //     values: [false, false]
-    //   }, {
-    //     id: 3,
-    //     title: 'Question Text 3',
-    //     values: [false, false]
-    //   }, {
-    //     id: 4,
-    //     title: 'Question Text 4',
-    //     values: [false, false]
-    //   }]
-    // })
+  describe('render', () => {
```

Comment:
why dont you use snapshots?

Reply:
Not sure what you mean, there's a snapshot test right below.